frontend/packages/ux-editor/src/containers/DesignView/PageGroupAccordion.test.tsx

```diff
@@ -0,0 +1,98 @@
+import type { IFormLayouts } from '../../types/global';
+import { layout1NameMock, layoutMock } from '../../testing/layoutMock';
+import { createQueryClientMock } from 'app-shared/mocks/queryClientMock';
+import { app, org, pageGroupAccordionEllipsisMenu } from '@studio/testing/testids';
+import { QueryKey } from 'app-shared/types/QueryKey';
+import userEvent from '@testing-library/user-event';
+import { textMock } from '@studio/testing/mocks/i18nMock';
+import type { ServicesContextProps } from 'app-shared/contexts/ServicesContext';
+
+const pagesMock: PagesModel = {
+  pages: null,
+  groups: [
+    {
+      name: 'Group 1',
+      order: [{ id: 'Side 1' }],
+    },
+    {
+      name: 'Group 2',
+      order: [{ id: 'Side 2' }],
+    },
+  ],
+};
+
+const layoutSetName = layoutSet1NameMock;
+const layouts: IFormLayouts = {
+  [layout1NameMock]: layoutMock,
+};
+
+describe('PageGroupAccordion', () => {
+  it('should disable move-up for first group, and move-down for last group', async () => {
+    await render({});
+    expect(moveGroupUpButton(0)).toBeDisabled();
+    expect(moveGroupDownButton(0)).toBeEnabled();
+    expect(moveGroupUpButton(1)).toBeEnabled();
+    expect(moveGroupDownButton(1)).toBeDisabled();
+  });
+
+  it('should correctly call mutation on changePageGroupOrder when moving a group up', async () => {
+    const user = userEvent.setup();
+    const changePageGroups = jest.fn();
+    await render({ queries: { changePageGroups } });
+    await user.click(moveGroupUpButton(1));
+    expect(changePageGroups).toHaveBeenCalledTimes(1);
+    const expectedPagesMock = pagesMock;
+    expectedPagesMock.groups = pagesMock.groups.toReversed();
+    expect(changePageGroups).toHaveBeenCalledWith(org, app, layoutSetName, expectedPagesMock);
+  });
+
+  it('should correctly call mutation on changePageGroupOrder when moving a group up', async () => {
+    const user = userEvent.setup();
+    const changePageGroups = jest.fn();
+    await render({ queries: { changePageGroups } });
+    await user.click(moveGroupDownButton(0));
+    expect(changePageGroups).toHaveBeenCalledTimes(1);
+    const expectedPagesMock = pagesMock;
+    expectedPagesMock.groups = pagesMock.groups.toReversed();
+    expect(changePageGroups).toHaveBeenCalledWith(org, app, layoutSetName, expectedPagesMock);
+  });
+});
+
+const groupAccordionHeader = (nth: number) =>
+  screen.getByTestId(pageGroupAccordionEllipsisMenu(nth));
+const moveGroupUpButton = (nth: number) =>
+  within(groupAccordionHeader(nth)).getByRole('button', {
+    name: textMock('ux_editor.page_menu_up'),
+  });
+const moveGroupDownButton = (nth: number) =>
+  within(groupAccordionHeader(nth)).getByRole('button', {
+    name: textMock('ux_editor.page_menu_down'),
+  });
+
+type renderParameters = {
+  props?: Partial<PageGroupAccordionProps>;
+  queries?: Partial<ServicesContextProps>;
+};
+
+const render = async ({ props, queries }: renderParameters) => {
```

Comment:
```suggestion
const renderPageGroupAccordion = async ({ props, queries }: renderParameters) => {
```

Can we rename this function to `renderPageGroupAccordion` to distinguish it from the built-in `render` function? 🙂